vivarium/core/experiment.py

```diff
@@ -1367,8 +1367,9 @@ def emit_configuration(self):
             'experiment_id': self.experiment_id,
             'name': self.experiment_name,
             'description': self.description,
-            'processes': serialize_dictionary(self.processes),
             'topology': self.topology,
+            # TODO -- handle large parameter sets in self.processes to meet mongoDB limit
+            # 'processes': serialize_dictionary(self.processes),
```

Comment:
Hmm, this might be tricky since we really do want to save all those parameters. Maybe we'll have to split them up across many documents?

Reply:
@U8NWXD -- this limit is normally sufficient. However, I ran some experiments with multiple agents that had entire E. coli genome sequences in their parameters. Maybe for now we could keep on emitting parameters, except for these large cases?